src/main/java/medicaldoctor/controllers/NavigationBarController.java

```diff
@@ -198,7 +198,7 @@ public void initialize(URL location, ResourceBundle resources) {
         //buttonMustHaveThisPermission(buttonMyLabRequests, Permission.ACCESS_MY_LAB_REQUESTS);
         buttonMustHaveThisPermission(buttonRegisterNewUser, Permission.REGISTER_NEW_USER);
         //buttonMustHaveThisPermission(buttonLabRequests, Permission.ACCESS_ALL_LAB_REQUESTS);
-        buttonMustHaveThisPermission(buttonLogs, Permission.ACCESS_LOGS);
+        //buttonMustHaveThisPermission(buttonLogs, Permission.ACCESS_LOGS);
```

Comment:
Will logs just be a trace of all things the user searched? Are we adding logs before we submit?

Reply:
imma say not enough time